libcxx/utils/ci/buildkite-pipeline.yml

```diff
@@ -728,6 +728,22 @@ steps:
           limit: 2
     timeout_in_minutes: 120
 
+  - label: "LLDB Data Formatters"
+    command: "libcxx/utils/ci/run-buildbot check-lldb-data-formatters"
+    artifact_paths:
+      - "**/test-results.xml"
+    env:
+        CC: "clang-${LLVM_HEAD_VERSION}"
+        CXX: "clang++-${LLVM_HEAD_VERSION}"
+    agents:
+      queue: "libcxx-builders"
+      os: "linux"
+    retry:
+      automatic:
+        - exit_status: -1  # Agent was lost
+          limit: 2
+    timeout_in_minutes: 120
```

Comment:
How much time does this job actually take to run? Because I don't think we have headroom for something that's close to 120 minutes (unless that counts the time spent waiting to run).

Reply:
On my M1 machine with a full debug build of clang/lldb/libc++ it took `77.92s`.

Reply:
The timeout we have on BuildKite jobs is the same everywhere, it's not really related to any job actually taking close to that amount of time. We basically set a timeout to avoid the case where an infinite loop would cause a job to never finish.